shared_files/cclf_BadFileNames/T.A0001.ACO.ZC8Y18.D181120.T1000009

```diff
@@ -0,0 +1,6 @@
+1A69B98CD30203031401M 25013010891940-10-271107710NA                              FnameOne                      A              LnameOne                                032016-08-012017-11-01Address L1b1                                 Address L2b1                                 Address L3b1                            Address L4b1                            Address L5b1                            Address L6b1                            CITY1                                                                                               MD211171111
```

Comment:
forgive me if this is a silly question, but did we generate the bad file names or did they give us a sample of bad files?

Reply:
We made them ourselves.